README.md

```diff
@@ -22,26 +22,25 @@ This allows you to test your plugin at every step.
 It's also recommended that you go through the [planning guide](meta_docs/planning-guide.md) before starting to develop your plugin.
 
 # Bootstrap a new Pulp plugin
-
+ 
 The first step is to bootstrap this template. This will create a functional but useless plugin,
-with minimal code, docs, and tests. Later on we'll discuss exactly what each part of this template does and what to
-change to create a 'real' plugin.
+with minimal code, docs, tests, and default Travis CI configuration. Later on we'll discuss
+exactly what each part of this template does and what to change to create a 'real' plugin.
 
 1. Clone this repository
 
    ``$ git clone https://github.com/pulp/plugin_template.git``
 
    ``$ cd plugin_template``
 
-2. Run the provided ``bootstrap.py`` script to create a skeleton for your plugin with the name of your choice.
+2. Run the provided ``plugin-template`` script to create a skeleton for your plugin with the name of your choice.
    It will contain a ``setup.py``, expected plugin layout and stubs for necessary classes and methods, minimal docs,
    and tests.
 
-   ``$ ./bootstrap.py --bootsrap your_plugin_name``
+   ``$ ./plugin-template --bootsrap plugin_name plugin_app_label``
 
-   **NOTE** : Whatever you choose for `your_plugin_name` will be prefixed with `pulp_`.
-   Therefore, for this argument it is best to just provide the content type
-   which you would like to support, e.g. `rubygem` or `maven`.
+   **NOTE** : The `plugin_app_label` should identify the content type which you would like to support, e.g. `rubygem` or `maven`.
+   The `plugin_name` is usually `pulp_` or `pulp-` prepended to the `plugin_app_label`, e.g. `pulp_maven`.
```

Comment:
Do we really want to give away that freedom? Also i do not see the benefit. I think we cannot have `pulp_perl` _and_ `pulp-perl` at the same time. Also, we do not want that, i guess.

Reply:
We already have a plugin that's called pulp-certguard[0]. I don't expect anyone to try to create pulp_certguard. I don't think the plugin template needs to guard against this. 

https://github.com/pulp/pulp-certguard/